omnibus/config/software/pip3.rb

```diff
@@ -22,4 +22,11 @@
   end
 
   command "#{python_bin} setup.py install --prefix=#{python_prefix}"
+
+  if ohai["platform"] != "windows"
+    block do
+      FileUtils.rm_f(Dir.glob("#{install_dir}/embedded/lib/python3.*/site-packages/pip-19.0.3-py3.*.egg/pip/_vendor/distlib/*.exe"))
+      FileUtils.rm_f(Dir.glob("#{install_dir}/embedded/lib/python3.*/site-packages/pip/_vendor/distlib/*.exe"))
```

Comment:
Any idea why this isn't there in python 2?

Reply:
I think the better question would be: why is this here in python 3? The `pip-19.0.3-py3.7.egg/pip` and `pip` folders have the same structure and same contents, so to me it seems like we are installing the same pip3 twice (with different methods).
I have no answer as to why this happens, but it shouldn't block this PR: we can investigate why we're shipping pip3 twice in another PR

Reply:
Could be that this is a symlink in the original installation, and at some point we make a copy following the symlink, so we end up with the same files twice (instead of once + a symlink). But I agree this is out of the scope of the PR.